doc/macos.md

```diff
@@ -62,19 +62,21 @@ Another alternative is to build a static version of r2 and just sign it.
 	$ sys/static.sh
 	$ make -C binr/radare2 osxsign
 
-You can verify that the binary is properly signed and verified by the code signing utility:
+You can verify that the binary is properly signed and verified by using the code signing utility:
 
 	$ codesign -dv binr/radare2/radare2
 
-After doing it you should be able to debug on OSX without root permissions!
+After doing it you should be able to debug on macOS without root permissions!
 
-	$ r2 -d ls
+	$ r2 -d r2
```

Comment:
why

Reply:
/bin/ls is signed by Apple!
```
$ which -a ls
ls: aliased to ls -G
/bin/ls
$ codesign -dvv /bin/ls
Executable=/bin/ls
Identifier=com.apple.ls
Format=Mach-O thin (x86_64)
CodeDirectory v=20100 size=381 flags=0x0(none) hashes=8+2 location=embedded
Platform identifier=2
Signature size=4105
Authority=Software Signing
Authority=Apple Code Signing Certification Authority
Authority=Apple Root CA
Info.plist=not bound
TeamIdentifier=not set
Sealed Resources=none
Internal requirements count=1 size=60
```
Trying to debug it:
```
$ r2 -d /bin/ls
Child killed
ptrace: Cannot attach: Invalid argument
Possibly unsigned r2. Please see doc/osx.md
ERRNO: 22 (EINVAL)
[w] Cannot open 'dbg:///bin/ls' for writing.
```
Debugging a binary not signed by Apple (eg. radare2):
```
$ r2 -d r2
= attach 38189 38189
bin.baddr 0x100000000
Using 0x100000000
asm.bits 64
 -- If you want to open the file in read-write mode, invoke r2 with '-w'
[0x100070000]> dc
Usage: r2 [-ACdfLMnNqStuvwzX] [-P patch] [-p prj] [-a arch] [-b bits] [-i file]
          [-s addr] [-B baddr] [-M maddr] [-c cmd] [-e k=v] file|pid|-|--|=

==> Process finished
```
The same also occurs using Apple's own debugger (lldb):
```
$ lldb ls
(lldb) target create "ls"
Current executable set to 'ls' (x86_64).
(lldb) r
error: process exited with status -1 (cannot attach to process due to System Integrity Protection)
$ lldb r2
(lldb) target create "r2"
Current executable set to 'r2' (x86_64).
(lldb) r
Process 38256 launched: '/usr/local/bin/r2' (x86_64)
Usage: r2 [-ACdfLMnNqStuvwzX] [-P patch] [-p prj] [-a arch] [-b bits] [-i file]
          [-s addr] [-B baddr] [-M maddr] [-c cmd] [-e k=v] file|pid|-|--|=
Process 38256 exited with status = 0 (0x00000000)
```

Reply:
Yes but that's a bad example just put r2 -d mybin instead.

Reply:
pls keep ls :P

Reply:
nah jk, r2 -d ls will not work on SIP, r2 -d r2 will work because of symlinks to home